runtime/gateway.go

```diff
@@ -131,6 +125,8 @@ func CreateGateway(
 		return nil, err
 	}
 
+	gateway.registerPredefined()
```

Comment:
If it did the linter would catch it (I've seen it catch other function calls that return error but are not assigned/handled). This line was moved.